tests/integration/hello_test.py

```diff
@@ -0,0 +1,76 @@
+import json
+import subprocess
+from threading import Thread
+from time import sleep, time
+
+from talker.client import get_talker
+
+
+def run_docker_compose():
+    print('running docker-compose...')
+    subprocess.Popen('docker-compose up', shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    print('docker compose is done')
+
+
+def get_redis_container_ip():
+    while True:
+        redis_container_id = subprocess.check_output(
+            "docker ps | grep redis| awk -F '    ' '{print $1}'",
+            shell=True).strip().decode('utf-8')
+
+        if redis_container_id:
+            inspect_res_str = subprocess.check_output('docker inspect {}'.format(redis_container_id), shell=True)
+            inspect_res = json.loads(inspect_res_str)
+            redis_container_ip = inspect_res[0]['NetworkSettings']['Networks']['integration_default']['IPAddress']
+            break
+        else:
+            print('waiting for redis container to be ready...')
+            sleep(3)
+    return redis_container_ip
+
+
+def wait_for_talker_container():
+    while True:
+        try:
+            subprocess.check_output('docker ps | grep talker', shell=True)
+            break
+        except subprocess.CalledProcessError:
+            print('waiting for talker container to be ready...')
+            sleep(1)
+
+
+def setup():
+    # clean all images and containers
+    try:
+        print('cleaning docker images and containers...')
+        p = subprocess.Popen(
+            'docker stop $(docker ps -aq); docker system prune -f; docker rmi  -f $(docker images -a -q)', shell=True,
```

Comment:
this is very destructive. I cant run this on my laptop because it will remove all of my containers and image (even not talker related)